app/Widgets/Intranet/Folder.js

```diff
@@ -50,41 +64,46 @@ export default class Folder extends React.Component {
   displayStructure = (obj) => {
     const params = {
       items: obj,
-      goToPath: this.props.goToPath,
+      goForward: this.props.goForward,
       showAttachment: this.showAttachment,
       pathString: this.props.pathString,
     };
     return (<ListItem {...params} />);
   }
 
   search = () => {
-    // console.log(Array.from(this.props.location.keys()).filter(e => e.includes(this.props.search)));
+      // console.log(Array.from(this.props.location.keys()).filter(e => e.includes(this.props.search)));
   }
   render() {
     const isDashboard = this.props.dashboard ?
-      (
-        <Link to={'intranet'} >
-          Intranet
-        </Link>
-      )
-      :
-      (
-        <Toolbar
-          pathString={this.props.pathString}
-          goToPath={this.props.goToPath}
-          timeStamp={this.props.timeStamp}
-          folders={this.props.location.count() || 0}
-          setSearch={this.props.setSearch}
-        />
-      );
+        (
+          <Link to={'intranet'} style={style.head}>
+            Intranet
+          </Link>
+        )
+        :
+        (
+          <Toolbar
+            pathString={this.props.pathString}
+            goToStringPath={this.props.goToStringPath}
+            timeStamp={this.props.timeStamp}
+            folders={this.props.location.count()}
+            setSearch={this.props.setSearch}
+          />
+        );
     return (
-        <div style={style.main} id="scroller">
-            <Scrollbars style={{ height: window.innerHeight - 50 }}>
```

Comment:
This is critical for displaying intranet correctly. 
I would recommend refactoring the code, 
and do the thing which you did only when this.props.dashboard is true.